### PR TITLE
Parse email even if bcc is an empty string

### PR DESCRIPTION
**Pull request by plicjo**

### CODE REVIEW

File: lib/griddler/sendgrid/adapter.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def remove_addresses_from_bcc(bcc, addresses)
       end
 
       def bcc_from_envelope(envelope)
-        JSON.parse(envelope)["to"] if envelope
+        JSON.parse(envelope)["to"] if !envelope.blank?
```
Comment (sdhull) [inline lib/griddler/sendgrid/adapter.rb]: Personally I prefer `foo.present?` over `!foo.blank?`... take it or leave it, your option :grin:

**Comment by sdhull** [comment]
:+1:   (assuming the `envelope` param is ever actually an empty string)

Sooooo @calebthompson does this gem no longer have an active maintainer? Seems like a few outstanding PRs without any activity indicates the project is on life-support... it's not really clear why the bcc PR got so much attention while the others have been ignored

File: lib/griddler/sendgrid/adapter.rb (hunk 1 above)
Comment (plicjo) [inline lib/griddler/sendgrid/adapter.rb]: `envelope.present?` will return true if `envelope` is an empty string. This PR is supposed to guard against both a nil envelope and an empty string envelope.

Comment (sdhull) [inline lib/griddler/sendgrid/adapter.rb]: `present?` is literally defined as `not blank?`. See:
```ruby
[1] pry(main)> show-source "foo".present?

From: /Users/stevehull/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22.1/lib/active_support/core_ext/object/blank.rb @ line 20:
Number of lines: 3
Owner: Object
Visibility: public

def present?
  !blank?
end
```

Comment (plicjo) [inline lib/griddler/sendgrid/adapter.rb]: Good call, I dropped the ball right there.

**Comment by sdhull** [comment]
The reason I ask is I have a few other changes I've needed to make to params work (fix character encodings, extract text from html when body is blank, etc). I'd be happy to submit these changes as PRs if they'll be considered. Otherwise I guess maybe I'll maintain my own fork.

**Comment by calebthompson** [comment]
Because someone pestered me, mostly. This gem doesn't really have an active maintainer since @gabebw and I passed the griddler proper gem off to another maintainer, but this PR looks fine to me.

**Comment by calebthompson** [comment]
https://github.com/thoughtbot/griddler-sendgrid/issues/10

**Pull request merged**